css/talk.styl

```diff
@@ -88,6 +88,9 @@
       color: white
       border-color: MAIN_HIGHLIGHT
 
+    &.create-discussion-or-comment
```

Comment:
How about we use the existing classes and add any CSS adjustments under the scoped `.subject-page-tabs` in the `tabbed-content.styl` file?